golem-component-service/db/migration/postgres/001__init.sql

```diff
@@ -1,13 +1,23 @@
 CREATE TABLE components
 (
-    component_id        uuid    NOT NULL,
+    namespace           text    NOT NULL,
+    component_id        uuid    NOT NULL PRIMARY KEY,
```

Comment:
It is common to keep primary key field on top.

Reply:
moved to top